web/components/token-swap-list.tsx

```diff
@@ -41,6 +41,15 @@ export const tokenList = [
     address: 'SP3DX3H4FEYZJZ586MFBS25ZW3HZDMEW92260R2PR',
     decimals: 8
   },
+  {
+    id: 4,
```

Comment:
increment?

Reply:
I'm guessing @philipdesmedt isn't done with it yet.
Also the logo path is wrong on L48. Thanks for  having a look though 👀